ref/feature/spnego/examples.adoc

```diff
@@ -0,0 +1,27 @@
+== Examples
+
+Open Liberty supports many common SPNEGO scenarios and configurations.
+
+=== Limit SPNEGO authentication 
+You can filter HTTP requests to require SPNEGO authentication for only certain requests, web applications, hosts, or user agents.
+
+The following example configures the following things:
```

Comment:
'The following example configures the following things:'
Maybe rephrase this sentence. How about 'the example provided configures the following things/points'?

Reply:
Yeah, I considered this too, but I don't think that I should use a slash between two words, so I've left as is.